examples/proposed/url-shortener.main.w

```diff
@@ -11,14 +11,14 @@ class UrlShortener {
   }
 
   // Generates a short id for the given url.
-  inflight getId(url: str): str {
+  pub inflight getId(url: str): str {
     let id = this._makeId();
     this.lookup.put(id, url);
     return id;
   }
 
   // Get the url for the given id. Returns nil if the id is invalid.
-  inflight getUrl(id: str): str? {
+  pub inflight getUrl(id: str): str? {
```

Comment:
Why the modifier change?

Reply:
Ah - this is code from the proposed examples folder. It's not getting compiled by CI but I just came around to updating the code